packages/boot/src/__tests__/acceptance/application-metadata.booter.acceptance.ts

```diff
@@ -11,7 +11,7 @@ import {BooterApp} from '../fixtures/application';
 describe('application metadata booter acceptance tests', () => {
   let app: BooterApp;
   const SANDBOX_PATH = resolve(__dirname, '../../.sandbox');
-  const sandbox = new TestSandbox(SANDBOX_PATH);
+  const sandbox = new TestSandbox(SANDBOX_PATH, '');
```

Comment:
Ditto.

Reply:
This one is tricky as we need to make sure `application.js` can still resolve its imports after being copied to the sandbox.